src/widget/scroll.rs

```diff
@@ -104,6 +126,69 @@ impl<T: Data> Scroll<T> {
         self.direction = ScrollDirection::Horizontal;
         self
     }
+
+    /// Draw scroll bars.
+    fn draw_bars(&self, paint_ctx: &mut PaintCtx, viewport: &Rect) {
+        if self.scroll_bars.opacity <= 0.0 {
+            return;
+        }
+
+        let brush = paint_ctx
+            .render_ctx
+            .solid_brush(SCROLL_BAR_BG.with_alpha(self.scroll_bars.opacity));
+        let border_brush = paint_ctx.render_ctx.solid_brush(SCROLL_BAR_BORDER);
```

Comment:
I think there's a little fine-tuning that can be done on the appearance. In particular, the border does not fade smoothly during the fade animation, but disappears suddenly at the end. This is pretty subtle because its opacity is low, but could be better. I think the right way to do that is to put both fill and stroke in an opacity group, but this is not in piet yet.

Reply:
Changed border color to have a dynamic opacity.